cheetah/accelerator/bpm.py

```diff
@@ -62,7 +61,7 @@ def plot(self, ax: plt.Axes, s: float, vector_idx: Optional[tuple] = None) -> No
 
     @property
     def defining_features(self) -> list[str]:
-        return super().defining_features
+        return super().defining_features + ["is_active"]
```

Comment:
For elements where `is_active` can be set manually, shouldn't it be part of `defining_properties`, i.e. wouldn't we want it to also be included in when lattices are saved as well?

Reply:
I had thought that I applied that change a few minutes ago. Maybe something went wrong with that

Reply:
Ah, I misunderstood. Is your suggestion to apply the same change to other elements with a manually set `is_active` property?